packages/react-data-grid/src/ChildRowDeleteButton.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+require('../../../themes/react-data-grid-cell.css');
```

Comment:
You only need to import the styles once in the parent component.
Is this supposed to be used without `Cell`?